cob_bringup/tools/docker_control.launch

```diff
@@ -32,4 +33,8 @@
 	<node pkg="cob_docker_control" type="docker_control.py" respawn="false" name="docker_control" output="screen" unless="$(arg multi_station)">
 	</node>
```

Comment:
There still is a `docker_control.py` in https://github.com/ipa320/cob_substitute/tree/indigo_dev/cob_docker_control/scripts, maybe it uses this one...
Do you have an overlay of `cob_substitute`?
Can you put one in your workspace and remove the `docker_control.py`? Then you should get an error....
Please provide a PR for cob_substitute where the script is removed...

In the same way, we might need to add a `fake_docking.py` to `cob_substitute`!